deployments/BridgeDeps.Dockerfile

```diff
@@ -2,9 +2,10 @@
 #
 # This image is meant to be used as a building block when building images for
 # the various components in the bridge repo, such as nodes and relayers.
-FROM ubuntu:xenial
+FROM ubuntu:20.04
 
 ENV LAST_DEPS_UPDATE 2020-12-21
```

Comment:
We should bump this and the one on line 21. 

The idea behind these lines is that we can use them to break the cache and update dependencies if needed.